valhalla/common/isochrones_core.py

```diff
@@ -126,22 +126,37 @@ def get_features(self, id_field_value, options={}):
         :rtype: QgsFeature
         """
 
-        features = [feature for feature in self.response['features'] if feature['geometry']['type'] in ('LineString', 'Polygon')]
-
+        features = [feature for feature in self.response['features'] if feature['geometry']['type'] in ('LineString', 'Polygon', 'MultiPolygon')]
         # Sort features based on the isochrone value, so that longest isochrone
         # is added first. This will plot the isochrones on top of each other.
         l = lambda x: x['properties']['contour']
         for isochrone in sorted(features, key=l, reverse=True):
             feat = QgsFeature()
             coordinates = isochrone['geometry']['coordinates']
+            geom_type = isochrone['geometry']['type']
             iso_value = isochrone['properties']['contour']
-            #metric = isochrone['properties']['metric']
-            if self.geometry == 'Polygon':
```

Comment:
`self.geometry` was set based on the `polygons` input type param, but we cannot rely on that anymore when determining the geom type, since it can be multipolygon

Reply:
they still don't have a convenience wrapper for geojson input? just curious, for this project I'd just do the minimum effort version:)

Reply:
Tried to find one but didn't